CHANGELOG.md

```diff
@@ -1,341 +1,379 @@
-# ROCm Performance Primitives Library
+# Changelog for RPP
 
-## Changelog
+Full documentation for RPP is available at (https://rocm.docs.amd.com/projects/rpp/en/latest/).
 
-### Online Documentation
-[RPP Documentation](https://gpuopen-professionalcompute-libraries.github.io/rpp/)
+### RPP 1.5.0 (unreleased)
 
-### RPP 1.5.0 (Unreleased)
+### Changes
 
-#### Added
-* 
+* Prerequisites
 
-#### Optimizations
-* 
+### Tested configurations
 
-#### Changed
-* Prerequisites 
-
-#### Fixed
-* 
-
-### Tested Configurations
 * Linux distribution
-  + Ubuntu - `20.04` / `22.04`
-  + CentOS - `7`
-  + RHEL - `8`/`9`
+  * Ubuntu - `20.04` / `22.04`
```

Comment:
Is this nested bullet format required for visualization? If not, can we leave the current format?

Reply:
Sorry, I'm not sure I understand your ask. The current version also has nested bullets, as the OSs listed fall under Linux distributions.

Reply:
I also think we need to remove the supported OS list here and apply ROCm support only

Reply:
@saadrahim some libraries only support a subset of ROCm supported OS. I think it is good to just list what the library supports.

Reply:
@LisaDelaney thanks for making all the changes. Our team uses the VS code markdown formatter and I think it defaults to using `*`/`+`/`-` for nested bullets. We can make that change to config file to stay consistent.

Reply:
@kiritigowda I prefer using different style bullets for different levels...sadly, our linter doesn't approve. :/